RealmSwift/SwiftUI.swift

```diff
@@ -426,6 +441,26 @@ private class ObservableStorage<ObservedType>: ObservableObject where ObservedTy
         self.filter = filter
         self.sortDescriptor = sortDescriptor
     }
+    /// :nodoc:
+    public init(_ type: ResultType.Type,
+                configuration: Realm.Configuration? = nil,
+                where: ((Query<ResultType>) -> Query<ResultType>)? = nil,
+                keyPaths: [String]? = nil,
+                sortDescriptor: SortDescriptor? = nil) {
+        self.storage = Storage(Results(RLMResults.emptyDetached()), keyPaths)
+        self.storage.configuration = configuration
+        self.where = `where`
+        self.sortDescriptor = sortDescriptor
+    }
+    /// :nodoc:
+    public init(_ type: ResultType.Type,
```

Comment:
Why adding this new initialiser?

Reply:
Without this the compiler will complain due to ambiguity.